src/removeTasks.js

```diff
@@ -0,0 +1,14 @@
+import { removeTask } from './taskManager.js';
+import { createItems } from './createItems.js';
+
+document.addEventListener('click', (event) => {
+  const clickedElement = event.target;
+  if (clickedElement.classList.contains('checking-box')) {
+    const listItem = clickedElement.parentNode;
+    const index = Array.from(listItem.parentNode.children).indexOf(listItem);
+    if (index !== -1) {
+      removeTask(index);
+      createItems();
+    }
+  }
```

Comment:
- You have done great so far, but there are some changes you need to make before moving to the next milestone. :+1: :+1: 
> Implement a function for deleting a task (remove an element from the array).

- [ ] You tried implementing the remove task functionality but your approach doesn't meet the project requirement, From your current implementation, a user needs to mark the checkbox and then click the `clear completed` to delete a task, but that's not the flow. You need to add a delete icon in front of each individual task and add an `onclick event listener` to delete the specific task. I'll suggest you implement this. :+1: :100: